psl/psl/tests/validation/attributes/index/missing_index_warning/relation_mode_prisma/one_field_at_at_unique.prisma

```diff
@@ -0,0 +1,23 @@
+// no relation index validation warning on relationMode = "prisma" when a referenced field is already in @@unique.
+
+generator client {
+    provider        = "prisma-client-js"
+    previewFeatures = ["referentialIntegrity"]
+}
+
+datasource db {
+    provider = "mysql"
+    url = env("TEST_DATABASE_URL")
+    relationMode = "prisma"
+}
+
+model SomeUser {
+    id Int @id
+    profile Profile?
+}
+
+model Profile {
+    id Int @id
+    userId Int @unique
+    user SomeUser? @relation(fields: [userId], references: [id])
+}
```

Comment:
I think this file is swapped with the next one now. The file name says "at_at_unique" but we have `@unique` here, whereas the other file is about single-field uniques, but contains an `@@unique`.